src/lib.rs

```diff
@@ -250,3 +250,18 @@ mod tests {
         }
     }
 }
+
+// This code exposes make_clockboard as a regular Javascript function. It returns a GeoJSON string.
+#[cfg(target_arch = "wasm32")]
+use wasm_bindgen::prelude::*;
+
+#[cfg(target_arch = "wasm32")]
+#[wasm_bindgen]
+pub fn make_clockboard(lat: f64, lon: f64, num_circles: usize, num_segments: usize) -> String {
```

Comment:
We might be able to directly pass in the `Params` struct and fill it out as a normal javascript object. Will keep playing around